flightsql/src/main/java/io/deephaven/server/flightsql/TableCreatorScopeTickets.java

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.server.flightsql;
+
+import io.deephaven.engine.table.Table;
+import io.deephaven.qst.TableCreator;
+import io.deephaven.qst.TableCreatorDelegate;
+import io.deephaven.qst.table.TicketTable;
+import io.deephaven.server.console.ScopeTicketResolver;
+import io.deephaven.server.session.SessionState;
+
+import java.nio.ByteBuffer;
+import java.util.Objects;
+
+final class TableCreatorScopeTickets extends TableCreatorDelegate<Table> {
+
+    private final ScopeTicketResolver scopeTicketResolver;
+    private final SessionState sessionState;
+
+    TableCreatorScopeTickets(TableCreator<Table> delegate, ScopeTicketResolver scopeTicketResolver,
+            SessionState sessionState) {
+        super(delegate);
+        this.scopeTicketResolver = Objects.requireNonNull(scopeTicketResolver);
+        this.sessionState = sessionState;
+    }
+
+    @Override
+    public Table of(TicketTable ticketTable) {
+        // This does not wrap in a nugget like TicketRouter.resolve; is that important?
```

Comment:
The purpose of the nuggets (there) is to capture gRPC initialization performance costs. Noting that `.resolve` may require cpu or network communication. I'd have to better understand the entry point to know if it is one that we already capture or if it is one we would also want to capture. As far as I can tell, it looks like you should be within the outer flight-sql ticket resolver's resolve call.

Adding a nugget here would enable a user to tell the difference between the flight-sql resolver and the scope ticket resolver taking excess time.

Reply:
This code path always get sourced from a getInfo, so I've wrapped getInfo with a nugget. Let me know if you think this is appropriate.